android/app/src/main/kotlin/coop/polypoly/polypod/polyOut/PolyOut.kt

```diff
@@ -15,11 +15,11 @@ open class PolyOut(
     private var statCache = mutableMapOf<String, MutableMap<String, String>>()
     companion object {
         private val fsPrefix = "polypod://"
-        fun idToPath(id: String, context: Context): String {
-            return context.filesDir.absolutePath + "/" + id.removePrefix(
+        fun idToPath(id: String, context: Context): String =
+            context.filesDir.absolutePath + "/" +
+                Preferences.currentFeatureName + "/" + id.removePrefix(
                 fsPrefix
-            )
-        }
+            ).removePrefix(Preferences.currentFeatureName)
```

Comment:
Er, we're removing prefixes twice (mind you, two different ones). Again, wouldn't it be better to have functions that map and unmap paths to polypod URIs and back?

Reply:
But this *is* a function that maps polypod URIs to file paths. Do you mean it could be named differently?